assets/js/script.js

```diff
@@ -0,0 +1,353 @@
+},
+//Question 8
+{ 
+  question: "What is sparkling wine in Spain called?", 
+  answers: ["Sekt", "Champagne", "Cava", "Prosecco"],
+  correctAnswer: "Cava",
+},
+//Question 9
+{ 
+  question: "How many gallons does the standard wine barrel contain?", 
+  answers: ["15", "30", "60", "100"],
+  correctAnswer: "60",
+},
+//Question 10
+{ 
+  question: "What is malolactic fermentation?", 
+  answers: ["Process in winemaking for converting wine into alcohol", "Process in winemaking for bottling", "Process in winemaking for siphoning wines from one tank or barrel to the next in hopes of leaving the precipitates and solids called pomace in the bottom of the tank", "Process in winemaking where tart malic acid from grape must is converted to softer and creamier latic acid"],
+  correctAnswer: "Process in winemaking where tart malic acid from grape must is converted to softer and creamier latic acid",
+},
+//Question 11
+{ 
+  question: "Where is champagne made?", 
+  answers: ["France", "Italy", "America", "All of the above"],
+  correctAnswer: "France",
+},
+//Question 12
+{ 
+  question: "Where is prosecco made?", 
+  answers: ["France", "Italy", "America", "All of the above"],
+  correctAnswer: "Italy",
+},
+//Question 13
+{ 
+  question: "Malbec originated in which country?", 
+  answers: ["France", "Italy", "Austria", "Spain"],
+  correctAnswer: "France",
+},
+//Question 14
+{ 
+  question: "Insert question 14 (?)", 
+  answers: ["a", "b", "c", "d"],
+  correctAnswer: "b",
+},
+//Question 15
+// { 
+//   question: "Insert question 15 (?)", 
+//   answers: ["a", "b", "c", "d"],
+//   correctAnswer: "b",
+// },
+// //Question 16
+// { 
+//   question: "Insert question 16 (?)", 
+//   answers: ["a", "b", "c", "d"],
+//   correctAnswer: "b",
+// },
+// //Question 17
+// { 
+//   question: "Insert question 17 (?)", 
+//   answers: ["a", "b", "c", "d"],
+//   correctAnswer: "b",
+// },
+// //Question 18
+// { 
+//   question: "Insert question 18 (?)", 
+//   answers: ["a", "b", "c", "d"],
+//   correctAnswer: "b",
+// },
+// //Question 19
+// { 
+//   question: "Insert question 19 (?)", 
+//   answers: ["a", "b", "c", "d"],
+//   correctAnswer: "b",
+// },
+// //Question 20
+// { 
+//   question: "Insert question 20 (?)", 
+//   answers: ["a", "b", "c", "d"],
+//   correctAnswer: "b",
+// },
+// //Question 21
+// { 
+//   question: "Insert question 21 (?)", 
+//   answers: ["a", "b", "c", "d"],
+//   correctAnswer: "b",
+// },
+// //Question 22
+// { 
+//   question: "Insert question 22 (?)", 
+//   answers: ["a", "b", "c", "d"],
+//   correctAnswer: "b",
+// },
+// //Question 23
+// { 
+//   question: "Insert question 23 (?)", 
+//   answers: ["a", "b", "c", "d"],
+//   correctAnswer: "b",
+// },
+// //Question 24
+// { 
+//   question: "Insert question 24 (?)", 
+//   answers: ["a", "b", "c", "d"],
+//   correctAnswer: "b",
+// },
+//Question 25
+{ 
+  question: "Insert question 25 (?)", 
+  answers: ["a", "b", "c", "d"],
+  correctAnswer: "b",
+},]
```

Comment:
Consider adding all of your questions to their own file. IE: `questions.json` which is one large json object with multiple questions. 

Ex. 

```json
{
    "1": {
        "question": "What is the volume of an average bottle of wine?",
        "answers": ["500mL", "750mL", "1000mL", "1 Gallon"],
        "correctAnswer": "750mL"
    },
    "2": {
        "question": "What are the five red varietals grown in the Bordeaux region?",
        "answers": [
            "Pinot Noir, Syrah, Cabernet Franc, Tempranillo, and Grenache",
            "Cabernet Sauvignon,  Cabernet Franc, Semillon, Merlot, and Sangiovese",
            "Cabernet Sauvignon, Sangiovese, Merlot, Mouvedre, and Gewürztraminer",
            "Cabernet Sauvignon, Merlot, Cabernet Franc, Petit Verdot, and Malbec"
        ],
        "correctAnswer": "Cabernet Sauvignon, Merlot, Cabernet Franc, Petit Verdot, and Malbec"
    }
}

```